[CRASH] - mid -playlist crash when using new widget

## Description
loaded up a 1000k playlist, somewhere around track 20 or so, it crashes. Trying to start again just crashed over and over until I noticed the widget looked weird. I was not using it to navigate so I deleted the widget and the app was able to start without crashing. 

## Steps to Reproduce
unable to reproduce but 

## Environment
 - Android device: Galaxy S20 FE 5g
 - Android OS version: Android 13 (One UI v5.1)
 - App version: 3.17.0


## Crash Logs/Stack trace
```  
2025-10-14 22:15:27.346 AndroidRuntime          com.cappielloantonio.tempo           E  FATAL EXCEPTION: main
                                                                                        Process: com.cappielloantonio.tempo, PID: 26642
                                                                                        g2.c: Unexpected exception thrown by non-Glide code
                                                                                        	at g2.m.run(SourceFile:79)
                                                                                        	at android.os.Handler.handleCallback(Handler.java:942)
                                                                                        	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                                        	at android.os.Looper.loopOnce(Looper.java:226)
                                                                                        	at android.os.Looper.loop(Looper.java:313)
                                                                                        	at android.app.ActivityThread.main(ActivityThread.java:8762)
                                                                                        	at java.lang.reflect.Method.invoke(Native Method)
                                                                                        	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
                                                                                        	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
                                                                                        Caused by: java.lang.IllegalArgumentException: RemoteViews for widget update exceeds maximum bitmap memory usage (used: 32001360, max: 20736000)
                                                                                        	at android.os.Parcel.createExceptionOrNull(Parcel.java:3049)
                                                                                        	at android.os.Parcel.createException(Parcel.java:3029)
                                                                                        	at android.os.Parcel.readException(Parcel.java:3012)
                                                                                        	at android.os.Parcel.readException(Parcel.java:2954)
                                                                                        	at com.android.internal.appwidget.IAppWidgetService$Stub$Proxy.updateAppWidgetIds(IAppWidgetService.java:950)
                                                                                        	at android.appwidget.AppWidgetManager.updateAppWidget(AppWidgetManager.java:626)
                                                                                        	at android.appwidget.AppWidgetManager.updateAppWidget(AppWidgetManager.java:723)
                                                                                        	at m3.a.k(SourceFile:98)
                                                                                        	at v2.f.l(SourceFile:161)
                                                                                        	at v2.f.k(SourceFile:100)
                                                                                        	at g2.m.run(SourceFile:63)
                                                                                        	at android.os.Handler.handleCallback(Handler.java:942) 
                                                                                        	at android.os.Handler.dispatchMessage(Handler.java:99) 
                                                                                        	at android.os.Looper.loopOnce(Looper.java:226) 
                                                                                        	at android.os.Looper.loop(Looper.java:313) 
                                                                                        	at android.app.ActivityThread.main(ActivityThread.java:8762) 
                                                                                        	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                        	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604) 
                                                                                        	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067) 
                                                                                        Caused by: android.os.RemoteException: Remote stack trace:
                                                                                        	at com.android.server.appwidget.AppWidgetServiceImpl.updateAppWidgetInstanceLocked(AppWidgetServiceImpl.java:2661)
                                                                                        	at com.android.server.appwidget.AppWidgetServiceImpl.updateAppWidgetIds(AppWidgetServiceImpl.java:2443)
                                                                                        	at com.android.server.appwidget.AppWidgetServiceImpl.updateAppWidgetIds(AppWidgetServiceImpl.java:2138)
                                                                                        	at com.android.internal.appwidget.IAppWidgetService$Stub.onTransact(IAppWidgetService.java:462)
                                                                                        	at android.os.Binder.execTransactInternal(Binder.java:1316)
2025-10-14 22:15:27.366 Process                 com.cappielloantonio.tempo           I  Sending signal. PID: 26642 SIG: 9
---------------------------- PROCESS ENDED (26642) for package com.cappielloantonio.tempo ----------------------------
```

## Screenshots
<img width="350" height="106" alt="Image" src="https://github.com/user-attachments/assets/fd44b188-582b-4d92-9643-571244311e08" />



## Reproducibility
somehow, I got into this state while being hands free and could not get out of the state. I have a lot of tracks downloaded and the app has about 6gb of data. Deleting the app cache did not remedy the issue but I will try deleting the app data from outside the app since the app will not start if the widget is loaded onto the screen.

--- a/app/src/main/java/com/cappielloantonio/tempo/widget/WidgetUpdateManager.java
+++ b/app/src/main/java/com/cappielloantonio/tempo/widget/WidgetUpdateManager.java
@@ -27,6 +27,8 @@ import java.util.concurrent.ExecutionException;
 
 public final class WidgetUpdateManager {
 
+    private static final int WIDGET_SAFE_ART_SIZE = 512;
+
     public static void updateFromState(Context ctx,
                                        String title,
                                        String artist,
@@ -95,7 +97,7 @@ public final class WidgetUpdateManager {
             CustomGlideRequest.loadAlbumArtBitmap(
                     appCtx,
                     coverArtId,
-                    com.cappielloantonio.tempo.util.Preferences.getImageSize(),
+                    WIDGET_SAFE_ART_SIZE,
                     new CustomTarget<Bitmap>() {
                         @Override
                         public void onResourceReady(Bitmap resource, Transition<? super Bitmap> transition) {

